src/block/Block.php

```diff
@@ -868,6 +868,12 @@ public function onEntityInside(Entity $entity) : bool{
 		return true;
 	}
 
+	/**
+	 * Called when an entity collide on this block
+	 */
+	public function onEntityCollide(Entity $entity, int $face) : void{
+	}
```

Comment:
you might want to add //NOOP since it's convention

Reply:
which convention ?

Reply:
> which convention ?

```php
	public function onEntityCollide(Entity $entity, int $face) : void{
	           //NOOP
	}
```